crates/node-core/src/metrics/version_metrics.rs

```diff
@@ -1,18 +1,40 @@
 //! This exposes reth's version information over prometheus.
 
-use crate::version::{build_profile_name, VERGEN_GIT_SHA};
+use crate::version::{ build_profile_name, VERGEN_GIT_SHA };
 use metrics::gauge;
 
-const LABELS: [(&str, &str); 6] = [
-    ("version", env!("CARGO_PKG_VERSION")),
-    ("build_timestamp", env!("VERGEN_BUILD_TIMESTAMP")),
-    ("cargo_features", env!("VERGEN_CARGO_FEATURES")),
-    ("git_sha", VERGEN_GIT_SHA),
-    ("target_triple", env!("VERGEN_CARGO_TARGET_TRIPLE")),
-    ("build_profile", build_profile_name()),
-];
+pub struct VersionInfo {
+    version: &str,
+    build_timestamp: &str,
+    cargo_features: &str,
+    git_sha: &str,
+    target_triple: &str,
+    build_profile: &str,
```

Comment:
can we make these pub?

if we extract metrics from node-core we'd need this as input I believe so makes sense to make these pub

Reply:
yeah it makes sense